coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -113,26 +117,50 @@ def _parse_documentation_with_symbols(self, param_identifiers,
 
             if stripped_line.startswith(param_identifiers[0]):
                 parse_mode = self.Parameter
+                # param_offset contains the starting column of param's name.
                 param_offset = line.find(
                     param_identifiers[0]) + len(param_identifiers[0])
+                # splitted contains the whole line from the param's name,
+                # which in turn is further divided into its name and desc.
                 splitted = line[param_offset:].split(param_identifiers[1], 1)
                 cur_param = splitted[0].strip()
 
                 param_desc = splitted[1]
+                # parsed section is added to the final list.
                 parsed.append(self.Parameter(name=cur_param, desc=param_desc))
 
+            elif stripped_line.startswith(excep_identifiers[0]):
+                parse_mode = self.ExceptionValue
+                excep_offset = line.find(
```

Comment:
variable names .. per prior comment ... and occurs many times in this patch.

(but `cur_` previous is ok for variable names if you prefer it, but it would be inappropriate for arg names, and unless there is a good reason `current_` should be used.)